server/models/SocialLink.ts

```diff
@@ -0,0 +1,80 @@
+import type { CreationOptional, InferAttributes, InferCreationAttributes, NonAttribute } from 'sequelize';
+
+import sequelize, { DataTypes, Model } from '../lib/sequelize';
+
+import Collective from './Collective';
+
+export enum SocialLinkType {
+  TWITTER = 'TWITTER',
+  TUMBLR = 'TUMBLR',
+  MASTODON = 'MASTODON',
+  MATTERMOST = 'MATTERMOST',
+  FACEBOOK = 'FACEBOOK',
+  INSTAGRAM = 'INSTAGRAM',
+  DISCORD = 'DISCORD',
+  YOUTUBE = 'YOUTUBE',
+  GITHUB = 'GITHUB',
+  GITLAB = 'GITLAB',
+  GIT_REPOSITORY = 'GIT_REPOSITORY',
+  OTHER = 'OTHER',
+}
```

Comment:
How was this list created?

Reply:
Websites I could remember lol
I know I am missing more big sites here, I will add to the list. We will be able to add more.

This is the top 20 currently used as collectives website urls:

```
        domain         | count
-----------------------+-------
 github.com            |  5174
 twitter.com           |  2181
 www.facebook.com      |   439
 www.linkedin.com      |   338
 www.aistechnolabs.com |   206
 www.meetup.com        |   169
 opencollective.com    |   160
 t.co                  |   157
 www.instagram.com     |   103
 linktr.ee             |    97
 www.youtube.com       |    92
 medium.com            |    91
 about.me              |    74
 bit.ly                |    62
 linkedin.com          |    57
 facebook.com          |    57
 t.me                  |    49
 discord.gg            |    46
 gitlab.com            |    40
 www.emccfrance.org    |    39
```

`repositoryUrl` is by large github.com

Reply:
Based on this analysis, maybe we can add linkedin, meetup.

Maybe `GIT_REPOSITORY` could be just `GIT`.

Not sure if MATTERMOST is relevant? If it is maybe we should have Slack too?